extras/benchmark/compile_nest.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -x
```

Comment:
Could you please add a docstring or rename to "compile_nest_on_jureca.sh" or similar?